CHANGELOG.asciidoc

```diff
@@ -13,9 +13,12 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha1...master[Check the HEAD d
 ==== Breaking changes
 
 *Affecting all Beats*
+
 - Add scripts for managing the dashboards of a single Beat {pull}1359[1359]
+- On DEB/RPM installations, the binary files are now found under `/usr/share/{{beat_name}}/bin`, not in `/usr/bin`. {pull}1385[1385]
```

Comment:
Symlinking might be confusing because without the correct `-path.*` parameters the Beats might behave quite different. An alternative idea is to have, say, a `filebeat.sh` that runs filebeat with the right parameters in foreground. Not sure how helpful that would be.